script/elevate-cpanel.PL

```diff
@@ -1038,6 +1038,11 @@ sub run_stage_2 ($self) {
 
     $self->elevation_startup_marker($sum);
 
+    # Remove scripts/upcp and bin/backup to make sure they don't
+    # end up running while ELevate is running.
+    $self->ssystem(qw{/usr/bin/rm -f /usr/local/cpanel/scripts/upcp});
+    $self->ssystem(qw{/usr/bin/rm -f /usr/local/cpanel/bin/backup});
```

Comment:
We ought to handle the case where these fail.  Perhaps ```ssystem_and_die()``` would suffice.

Reply:
Could we just do `unlink('/LC/bin/backup')` here please? There are far fewer things that can go wrong with that.